### ISSUE TITLE
Update README to explain new expected location of scripts/modules

### ISSUE THREAD

**Issue by malleoz** [TASLabz/dolphin#13]
[PR#10](https://github.com/TASLabz/dolphin/pull/10) aims to search for scripts/modules in Load/Scripts folder. We need to make this apparent in the README, along with any other script-related changes, such as the toggle feature we are adding.

- Also need to explain toggle functionality

**Comment by ximk** [comment]
It would probably be best to wait on this until we're satisfied with scripting changes on the backend and do a full rewrite afterwards.

**Comment by malleoz** [comment]
Yeah I agree with that, otherwise we risk accidentally leaving in logic which has changed at some point.
